### PR TITLE
Using the valence host to build the navigation link instead of last D…

### PR DESCRIPTION
**Pull request by BraydonKains**
…2L page

According to the following Slack conversation https://d2l.slack.com/archives/C0PHG3QB0/p1583264038087000 the `valenceHost` property is built using the WebServerName. According to this BI report 
image
the WebServerName config variables are set to exactly what we need on SBG's instances. A couple other devs I've discussed with seem to concur that the `valenceHost` is the way to go for this as well.

### CODE REVIEW

File: src/components/cpd-widget.js [hunk 1]
```diff
@@ -137,10 +136,8 @@ class CpdWidget extends BaseMixin(LitElement) {
 			.then((data) =>
 				this.subjects = data
 			);
-		this.cpdService.ParentHost(CpdRoutes.CpdHome())
-			.then(url => {
-				this.cpdLink = url;
-			});
+		this.cpdLink = this.cpdService.CpdHomeLink;
+		console.log(this.cpdLink);
```
Comment (TomMichielsD2L) [inline src/components/cpd-widget.js]: whoops

File: src/services/cpd-service.js [hunk 2]
```diff
@@ -198,13 +199,7 @@ export class CpdService {
...
 	static get JobTitle() { return 'target/jobtitles'; }
 	static get Method() { return 'method'; }
-	static ParentHost(route) {
-		return window.data.fraSettings.navigation.getLastD2LPage()
-			.then(page => {
-				const host = (new URL(page.url)).host;
-				return `http://${host}${route ? route : ''}`;
-			});
-	}
+
```
Comment (TomMichielsD2L) [inline src/services/cpd-service.js]: Do we need this empty line?

File: src/components/cpd-widget.js (hunk 1 above)
Comment (BraydonKains) [reply src/components/cpd-widget.js]: eslint why hath thou forsaken me

File: src/services/cpd-service.js (hunk 2 above)
Comment (BraydonKains) [reply src/services/cpd-service.js]: I would love to selfishly say yes (I hate not having lines between methods) but for consistency's sake, no.

File: src/components/cpd-widget.js (hunk 1 above)
Comment (jwong-d2l) [reply src/components/cpd-widget.js]: we should check that.

**Pull request merged**